apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/oversendelsebrev/OversendelseBrevRoute.kt

```diff
@@ -0,0 +1,71 @@
+package no.nav.etterlatte.brev.oversendelsebrev
+
+import io.ktor.http.HttpStatusCode
+import io.ktor.server.application.call
+import io.ktor.server.response.respond
+import io.ktor.server.routing.Route
+import io.ktor.server.routing.get
+import io.ktor.server.routing.post
+import io.ktor.server.routing.route
+import no.nav.etterlatte.brev.BREV_ID_CALL_PARAMETER
+import no.nav.etterlatte.brev.brevId
+import no.nav.etterlatte.brev.hentinformasjon.Tilgangssjekker
+import no.nav.etterlatte.libs.common.BEHANDLINGID_CALL_PARAMETER
+import no.nav.etterlatte.libs.common.sakId
+import no.nav.etterlatte.libs.common.withBehandlingId
+import no.nav.etterlatte.libs.common.withSakId
+import no.nav.etterlatte.libs.ktor.brukerTokenInfo
+import org.slf4j.LoggerFactory
+import kotlin.time.DurationUnit
+import kotlin.time.measureTimedValue
+
+private val logger = LoggerFactory.getLogger("oversendelseBrevRoute")
+
+fun Route.oversendelseBrevRoute(
+    tilgangssjekker: Tilgangssjekker,
+    service: OversendelseBrevService,
+) {
+    route("/brev/behandling/{$BEHANDLINGID_CALL_PARAMETER]/oversendelse") {
+        get {
+            withBehandlingId(tilgangssjekker) { behandlingId ->
+                measureTimedValue {
+                    service.hentOversendelseBrev(behandlingId)
+                }.let { (brev, varighet) ->
+                    logger.info("Henting av brev tok ${varighet.toString(DurationUnit.SECONDS, 2)}")
+                    if (brev != null) {
+                        call.respond(brev)
+                    } else {
+                        call.respond(HttpStatusCode.NoContent)
+                    }
+                }
+            }
+        }
+
+        post {
+            withBehandlingId(tilgangssjekker, skrivetilgang = true) { behandlingId ->
+                measureTimedValue {
+                    service.opprettOversendelseBrev(behandlingId, sakId, brukerTokenInfo)
+                }.let { (brev, varighet) ->
+                    logger.info("Oppretting av oversendelsebrev tok ${varighet.toString(DurationUnit.SECONDS, 2)}")
+                    call.respond(brev)
+                }
+            }
+        }
+    }
+
+    route("brev/{$BREV_ID_CALL_PARAMETER}/oversendelse") {
+        post("ferdigstill") {
+            withSakId(tilgangssjekker, skrivetilgang = true) { sakId ->
+                val brev = service.ferdigstillOversendelseBrev(brevId, sakId, brukerTokenInfo)
+                call.respond(brev)
+            }
+        }
+
+        post("distribuer") {
+            withSakId(tilgangssjekker, skrivetilgang = true) { sakId ->
+                val brev = service.distribuerOversendelseBrev(brevId, sakId, brukerTokenInfo)
+                call.respond(brev)
+            }
+        }
```

Comment:
Hvorfor ikke bruke `BrevRoute` sitt endepunkt for distribusjon?

Reply:
Det er kanskje bedre ja. Jeg tenker jeg heller går for den veien